lib/config.js

```diff
@@ -333,7 +333,11 @@ function Config(options) {
 
     if (useConfig) {
         debug("Using command line config " + useConfig);
-        this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
+        try {
+            this.useSpecificConfig = loadConfig(path.resolve(process.cwd(), useConfig));
```

Comment:
You can check to see if the first character is a letter, number, or underscore to determine if it's a package name out not. Avoid try-catch here.